addon/index.ts

```diff
@@ -7,5 +7,5 @@ export {
   didTransition,
   whenRoutePainted,
   whenRouteIdle,
-  emitMark,
+  beginScheduledWork,
```

Comment:
I don't think this needs to be exported, we don't use it in tests and I don't think users should use it right?